src/main/io/beeper.c

```diff
@@ -55,13 +55,18 @@ PG_REGISTER_WITH_RESET_TEMPLATE(beeperDevConfig_t, beeperDevConfig, PG_BEEPER_DE
 #endif
 #ifdef BEEPER
 #define BEEPER_PIN      BEEPER
+#define BEEPER_PWM_FREQ 0
+#elif defined(BEEPER_PWM)
+#define BEEPER_PIN      BEEPER_PWM
```

Comment:
@ledvinap: I suggested to use `BEEPER` (like for existing non-PWM beepers) to indicate the beeper pin, and add `BEEPER_IS_PWM` to indicate that it's a PWM beeper. This way, the ifdefs can be simplified.